docs/content/get-started/set-up.md

```diff
@@ -1,9 +1,10 @@
 # Get started: 1 - Set up
-There are three parts to this step: 
+There are four parts to this step: 
 
 * Sign up for a SpatialOS account (or make sure you are logged in)
 * Set up your machine
 * Get the GDK and the FPS Starter Project source code
+* Open the FPS Starter Project in your Unity Editor
 
 (This step is the longest of the 6 steps - the others are much quicker.)
```

Comment:
But now there are only 5 steps, and there are only 4 steps on this page? Which steps are we talking about? 
```suggestion
Note: Set up is the longest part of this get started guide. The other parts of this guide are much quicker
```
Or something? I'm not sure I like this line at all if I'm being honest.

Reply:
Good catch. See this and rejigging slightly.
(Reason for it is: Getting starting with the GDK is marketed as quick and simple...but the first thing you hit is this whopper of a step which is potentially off-putting. From memory, we popped in the NavMesh step temporarily.)